<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="BTN_Ok">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>280</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_Close">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>280</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_Default">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Default</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabSettings">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>445</width>
     <height>261</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_Main">
    <attribute name="title">
     <string>Main</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>421</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Path to market logs</string>
     </property>
     <widget class="QLineEdit" name="LE_Path">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>351</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QToolButton" name="BTN_ChangePath">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>171</width>
       <height>121</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Scan settings</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <widget class="QSpinBox" name="SB_AlarmTime">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="wrapping">
       <bool>false</bool>
      </property>
      <property name="specialValueText">
       <string/>
      </property>
      <property name="accelerated">
       <bool>false</bool>
      </property>
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>3600</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Alarm every (sec)</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="CB_AlarmEnable">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Tray alarm enable</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="CB_LoopScan">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Loop scan</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="CB_AlarmWithSound">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Sound alarm</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_SellOrders">
    <attribute name="title">
     <string>Table settings</string>
    </attribute>
    <widget class="QListWidget" name="List_TableSettings_Off">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>160</width>
       <height>220</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="List_TableSettings_On">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>160</width>
       <height>220</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="BTN_TableSettings_SetOn">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>130</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;&gt;&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BTN_TableSettings_SetOff">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>160</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;&lt;&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BTN_TableSettings_Up">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Up</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BTN_TableSettings_Down">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Down</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
